src/main/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpoint.java

```diff
@@ -109,7 +119,18 @@ public GlobalSearchResponsePayload searchForCases(@RequestBody @Valid GlobalSear
         log.debug("GlobalSearchEndpoint.searchForCases execution completed in {} milliseconds...", between.toMillis());
 
         return result;
+    }
+
+    public static String buildCaseIds(GlobalSearchResponsePayload globalSearchResult) {
+        return globalSearchResult.getResults().stream().limit(MAX_CASE_IDS_LIST)
+            .map(c -> String.valueOf(c.getCaseReference()))
+            .collect(Collectors.joining(CASE_ID_SEPARATOR));
+    }
 
+    public static String buildCaseTypeIds(GlobalSearchResponsePayload globalSearchResult) {
+        return globalSearchResult.getResults().stream().limit(MAX_CASE_IDS_LIST)
```

Comment:
If we are logging the result caseTypeIds; should we create a distinct list first or continue taking the first 10?

Reply:
Resolving comment as statement no longer relevant after changes in cfb129896c8f8d8d4c56b4f0af2222221d6c5474